src/panel/poi/PoiPanel.jsx

```diff
@@ -218,6 +230,11 @@ export default class PoiPanel extends React.Component {
       return null;
     }
 
+    // Show PoI name in search field
+    SearchInput.setInputValue(poi.name);
```

Comment:
This line may be the only thing not easily moveable to PanelManager, as the POI is not known by it… which is a recurring problem that calls for a centralized current POI state upstream.
I think it's ok to keep it here for now.